src/components/Heroes/index.jsx

```diff
@@ -57,15 +59,18 @@ class RequestLayer extends React.Component {
         <div>
           <Heading
             title={title}
-            subtitle={`${abbreviateNumber(matchCount)} ${strings.hero_this_month}`}
+            subtitle={`${abbreviateNumber(matchCount)} ${
+              strings.hero_this_month
```

Comment:
Currently this string says "in the last 30 days" but for turbo it's "this month".

we'll probably need to add a new string and switch between them depending on tab.